src/fswAlgorithms/orbitControl/lambertPlanner/lambertPlanner.cpp

```diff
@@ -75,3 +75,55 @@ void LambertPlanner::readMessages()
     this->r_N = cArray2EigenVector3d(navTransInMsgBuffer.r_BN_N);
     this->v_N = cArray2EigenVector3d(navTransInMsgBuffer.v_BN_N);
 }
+
+/*! This method integrates the provided equations of motion using Runge-Kutta 4 (RK4) and returns the time steps and state vectors at each time step.
+    @param EOM equations of motion function to be propagated
+    @param interval integration interval
+    @param X0 initial state
+    @param dt time step
+    @return std::pair<std::vector<double>, std::vector<Eigen::VectorXd>>
+*/
+std::pair<std::vector<double>, std::vector<Eigen::VectorXd>> LambertPlanner::propagator(const std::function<Eigen::VectorXd(double, Eigen::VectorXd)>& EOM, std::array<double, 2> interval, const Eigen::VectorXd& X0, double dt)
```

Comment:
:large_orange_diamond:  should this be named `propagate` ?

Reply:
Done.